plugins/modules/mysql_replication.py

```diff
@@ -131,6 +131,11 @@
       L(MySQL encrypted replication documentation,https://dev.mysql.com/doc/refman/8.0/en/replication-solutions-encrypted-connections.html).
     type: str
     aliases: [master_ssl_cipher]
+  primary_ssl_verify_server_cert:
+    description:
+    - Same as mysql variable.
+    type: bool
+    aliases: [master_ssl_verify_server_cert]
```

Comment:
```suggestion
```
the other aliases exist for backwards compatibility, so no need to have it for new arguments